apps/web/src/services/dashboards/widgets/_hooks/use-widget-state.ts

```diff
@@ -200,6 +202,19 @@ export function useWidgetState<Data = any>(
             state.widgetConfig?.options_schema?.schema,
             dashboardDetailState.variablesSchema,
         )),
+        nonInheritOptionsTooltip: computed<string|undefined>(() => {
+            if (!state.widgetInfo?.schema_properties?.length) return undefined;
+            const enabledInheritOptions: string[] = Object.entries(state.widgetInfo.inherit_options).filter(([, v]) => v.enabled).map(([k]) => k);
+            const nonInheritOptions: string[] = [];
+            state.widgetInfo.schema_properties.forEach((property) => {
+                if (!enabledInheritOptions.includes(property)) nonInheritOptions.push(property);
+            });
+            if (!nonInheritOptions.length) return undefined;
+            const tooltipText = nonInheritOptions.map((d) => `<p>• ${getWidgetOptionsSchemaPropertyName(d)}</p>`).join('\n');
+            // song-lang
```

Comment:
🥲

Reply:
🥲222
I have set a flag to translate later all at once.